Chessington.GameEngine/Pieces/Pawn.cs

```diff
@@ -5,23 +5,27 @@ namespace Chessington.GameEngine.Pieces
 {
     public class Pawn : Piece
     {
+        
+        
         public Pawn(Player player) 
             : base(player) { }
 
         public override IEnumerable<Square> GetAvailableMoves(Board board)
         {
             var currentPosition = board.FindPiece(this);
             List<Square> availableMoves = new List<Square>();
-            Square square;
+            
             if (Player == Player.White)
             {
-                square = new Square(currentPosition.Row - 1, currentPosition.Col);
+                availableMoves.Add(new Square(currentPosition.Row - 1, currentPosition.Col));
+                if (currentPosition.Row == 7) availableMoves.Add(new Square(currentPosition.Row -2, currentPosition.Col));
```

Comment:
even if it is just one lines it looks consistent and neater if you use same formatting
if (......)
{
....
}

Reply:
and don't forget the space after - sign ;)